ext/build_defaults.yaml

```diff
@@ -17,10 +17,10 @@ build_dmg: TRUE
 msi_name: 'puppet-agent'
 build_msi:
   puppet_for_the_win:
-    ref: '59031fc0783c70706d1faa774ce4c9cfd74e2965'
+    ref: '1387b2afc02826c036d500c365cbc06235255439'
```

Comment:
Let's change this to `52e81967253924428bb0b1e7e1a11e8678d68378` for now.. and leave MCO out of this PR until we can get sorted on https://github.com/puppetlabs/puppet_for_the_win/pull/89/files